base_rest/components/service.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+import logging
+
+from odoo.addons.component.core import AbstractComponent
+from odoo.exceptions import UserError
+from odoo.http import request
+from odoo.tools.translate import _
+
+from werkzeug.exceptions import NotFound
+
+
+_logger = logging.getLogger(__name__)
+
+
+try:
+    from cerberus import Validator
+except ImportError:
+    _logger.debug('Can not import cerberus')
+
+
+def to_int(val):
+    # The javascript VM ducktape only use float and so pass float
+    # to the api, the werkzeug request interpret params as unicode
+    # so we have to convert params from string to float to int
+    if val:
+        return int(float(val))
+    else:
+        return None
+
+
+def to_bool(val):
+    return val in ('true', 'True', '1', True)
```

Comment:
Do we add **Y** value for system use char(1) as Boolean value ?
https://www.devart.com/dotconnect/oracle/docs/CharAsBoolean.html

Reply:
@kryskool Thank you for your comment. This module is a WIP. But I'll take into account your comment. The API is not yet finalized see https://github.com/OCA/rest-framework/issues/2

Reply:
@lmignon thanks for the feedback
i take a look at #2 
Regards,